python_modules/dagster/dagster/core/definitions/decorators/solid_decorator.py

```diff
@@ -56,6 +56,11 @@ def has_var_kwargs(self) -> bool:
         # var keyword arg has to be the last argument
         return len(params) > 0 and param_is_var_keyword(params[-1])
 
+    def get_output_annotation(self) -> Any:
+        from ..inference import infer_output_props
+
+        return infer_output_props(self.decorated_fn).annotation
+
```

Comment:
This allows us to refer back to the annotation of the underlying function from the definition itself.